Api/Data/Models/Thread.cs

```diff
@@ -0,0 +1,21 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace SharpScape.Api.Models;
+// cant use Thread as classname cause its already used in namespace. So I name it ThreadModel
+public class ThreadModel
+{
+    [Key]
+    public Guid Id{get;set;}
+    [ForeignKey("User")]
+    public Guid UserId {get;set;}
+    public User User{get;set;}
```

Comment:
can you run it or not? cuz I dont build I just use dotnet run then get error when using Thread thats why I have to change to ThreadModel something. I know the stuff in the database.

Reply:
There is no create and update actions in the controller right now because its not in  the task descriptions.

Reply:
That's true.

Also I can run the project with the model named `Thread`

Reply:
ok then let me commit these changes and run it

Reply:
about "why is a User object a member of ThreadModel?" as far as I know that the requirement when add foreignkey.

Reply:
It's not required, the `Thread` just needs to know the foreign Id.  When the frontend gets the `UserId` from the Dto, it can HTTP request `/api/user/21658723487` to find out who posted the thread

Reply:
its required for the the migration to make the sql commands to create foreignkey link in the database.
im not talking about the frontend.  
About the Thread.cs this is what I have when adding dbset
https://cdn.discordapp.com/attachments/824772845078577202/987447490297102396/unknown.png